### ISSUE TITLE
Systemd gen sets wrong path

### ISSUE THREAD

**Issue by ash-development** [hrfee/jfa-go#350]
#### Read the [FAQ](https://wiki.jfa-go.com/docs/faq/) first!

**Describe the bug**

When generating a systemd file, it gets the ExecStart to an invalid path where the executeable is not
`ExecStart=/root/.config/jfa-go`

**To Reproduce**

What to do to reproduce the problem.

**Logs**

```
root@ASHubuntu:~/jfa# systemctl --user start jfa-go
root@ASHubuntu:~/jfa# systemctl --user status jfa-go
× jfa-go.service - An account management system for Jellyfin.
     Loaded: loaded (/root/.config/systemd/user/jfa-go.service; disabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Tue 2024-07-30 04:07:00 EDT; 3s ago
    Process: 2173283 ExecStart=/root/jfa/jfa-go -d /root/.config/jfa-go (code=exited, status=203/EXEC)
   Main PID: 2173283 (code=exited, status=203/EXEC)
        CPU: 1ms

Jul 30 04:07:00 ASHubuntu systemd[736513]: Started An account management system for Jellyfin..
Jul 30 04:07:00 ASHubuntu systemd[2173283]: jfa-go.service: Failed to locate executable /root/jfa/jfa-go: No such file or directory
Jul 30 04:07:00 ASHubuntu systemd[2173283]: jfa-go.service: Failed at step EXEC spawning /root/jfa/jfa-go: No such file or directory
Jul 30 04:07:00 ASHubuntu systemd[736513]: jfa-go.service: Main process exited, code=exited, status=203/EXEC
Jul 30 04:07:00 ASHubuntu systemd[736513]: jfa-go.service: Failed with result 'exit-code'.
```

**Configuration**

I don't believe anything was configured

**Platform/Version**

Ubuntu 22.04, JFA Version 0.5.1

**Comment by hrfee** [comment]
Would you mind giving the exact command you ran to generate the file, and the intended path? everything in the systemd output looks good to me.

**Comment by ash-development** [comment]
@hrfee i ran `jfa-go systemd`

i apologize but i do not have the original file. upon trying again, it seems to set the `ExecStart` variable to whatever path you're in, inside a `jfa-go` folder.

running in `/root/jfa-go`
```service
ExecStart=/root/jfa-go/jfa-go
```
```cmd
root@ASHubuntu:~/jfa-go# cd jfa-go
-bash: cd: jfa-go: No such file or directory
```

running in `/mnt/jelly-vol/`
```service
ExecStart=/mnt/jelly-vol/jfa-go
```
```cmd
root@ASHubuntu:/mnt/jelly-vol# cd jfa-go
-bash: cd: jfa-go: No such file or directory
```

and when moving to `~/.config/systemd/user`, it provides the same error as provided in my original issue
```cmd
root@ASHubuntu:/mnt/jelly-vol# systemctl --user status jfa-go
× jfa-go.service - An account management system for Jellyfin.
     Loaded: loaded (/root/.config/systemd/user/jfa-go.service; disabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Wed 2024-07-31 22:15:31 EDT; 2s ago
    Process: 2232992 ExecStart=/mnt/jelly-vol/jfa-go (code=exited, status=203/EXEC)
   Main PID: 2232992 (code=exited, status=203/EXEC)
        CPU: 1ms

Jul 31 22:15:31 ASHubuntu systemd[736513]: Started An account management system for Jellyfin..
Jul 31 22:15:31 ASHubuntu systemd[2232992]: jfa-go.service: Failed to locate executable /mnt/jelly-vol/jfa-go: No such file or directory
Jul 31 22:15:31 ASHubuntu systemd[2232992]: jfa-go.service: Failed at step EXEC spawning /mnt/jelly-vol/jfa-go: No such file or directory
Jul 31 22:15:31 ASHubuntu systemd[736513]: jfa-go.service: Main process exited, code=exited, status=203/EXEC
Jul 31 22:15:31 ASHubuntu systemd[736513]: jfa-go.service: Failed with result 'exit-code'.
```

and if i make the directory, it still provides the same error
```cmd
root@ASHubuntu:/mnt/jelly-vol# systemctl --user status jfa-go
× jfa-go.service - An account management system for Jellyfin.
     Loaded: loaded (/root/.config/systemd/user/jfa-go.service; disabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Wed 2024-07-31 22:15:59 EDT; 1s ago
    Process: 2234428 ExecStart=/mnt/jelly-vol/jfa-go (code=exited, status=203/EXEC)
   Main PID: 2234428 (code=exited, status=203/EXEC)
        CPU: 1ms

Jul 31 22:15:59 ASHubuntu systemd[736513]: Started An account management system for Jellyfin..
Jul 31 22:15:59 ASHubuntu systemd[2234428]: jfa-go.service: Failed to locate executable /mnt/jelly-vol/jfa-go: Is a directory
Jul 31 22:15:59 ASHubuntu systemd[2234428]: jfa-go.service: Failed at step EXEC spawning /mnt/jelly-vol/jfa-go: Is a directory
Jul 31 22:15:59 ASHubuntu systemd[736513]: jfa-go.service: Main process exited, code=exited, status=203/EXEC
Jul 31 22:15:59 ASHubuntu systemd[736513]: jfa-go.service: Failed with result 'exit-code'.
```

the only way that i can get this error to go away is to set the service file to
```service
ExecStart=/usr/bin/jfa-go
```

and once i do that, it starts. edited config & data directories because one already existed lol, in this case command was `jfa-go systemd -d /root/.config/NEWjfa -c /root/.config/NEWjfa/NEWconfig.ini`
```cmd
root@ASHubuntu:~/.config# systemctl --user status jfa-go
● jfa-go.service - An account management system for Jellyfin.
     Loaded: loaded (/root/.config/systemd/user/jfa-go.service; disabled; vendor preset: enabled)
     Active: active (running) since Wed 2024-07-31 22:20:16 EDT; 3s ago
   Main PID: 2247803 (jfa-go)
      Tasks: 13 (limit: 18595)
     Memory: 23.9M
        CPU: 82ms
     CGroup: /user.slice/user-0.slice/user@0.service/app.slice/jfa-go.service
             └─2247803 /usr/bin/jfa-go -d /root/.config/NEWjfa -c /root/.config/NEWjfa/NEWconfig.ini

Jul 31 22:20:16 ASHubuntu systemd[736513]: Started An account management system for Jellyfin..
Jul 31 22:20:16 ASHubuntu jfa-go[2247803]: jfa-go version: 0.5.1 (46566fb9)
Jul 31 22:20:16 ASHubuntu jfa-go[2247803]: Socket: /tmp/jfa-go.sock
Jul 31 22:20:16 ASHubuntu jfa-go[2247803]: 2024/07/31 22:20:16 Using internal storage
Jul 31 22:20:16 ASHubuntu jfa-go[2247803]: [INFO] 22:20:16  Copied default configuration to "/root/.config/NEWjfa/NEWconfig.ini"
Jul 31 22:20:16 ASHubuntu jfa-go[2247803]: [INFO] 22:20:16  Initializing router
Jul 31 22:20:16 ASHubuntu jfa-go[2247803]: [INFO] 22:20:16  Loading routes
Jul 31 22:20:16 ASHubuntu jfa-go[2247803]: [INFO] 22:20:16  Loading setup @ 0.0.0.0:8056
Jul 31 22:20:16 ASHubuntu jfa-go[2247803]: [INFO] 22:20:16  Loaded, visit 0.0.0.0:8056 to start.
```

i do not know if it is helpful information, but i did install with apt through the directions provided in the README

**Comment by hrfee** [comment]
sorry for the late reply, but wow I don't know I haven't caught this before. I't evaluates the executable name in the context of where the user started it from, so if you run `jfa-go systemd` while in `/tmp`, it's gonna think `jfa-go` is in `/tmp`, but if i ran `/usr/bin/jfa-go`, all would be good as it's an explicit path. I'll get it fixed now, thank you for reporting this in detail.

**Comment by hrfee** [comment]
Fixed in 8f3c723, I wrote in the commit that it fixed #352 by accident, but that's unrelated.

**Comment by ash-development** [comment]
> sorry for the late reply, but wow I don't know I haven't caught this before. I't evaluates the executable name in the context of where the user started it from, so if you run `jfa-go systemd` while in `/tmp`, it's gonna think `jfa-go` is in `/tmp`, but if i ran `/usr/bin/jfa-go`, all would be good as it's an explicit path. I'll get it fixed now, thank you for reporting this in detail.

yes ofc!! happy to help ^~^
